### PR TITLE
ISSUE-27 Scenario for checking UID FETCH command on 1000 messages

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by aduprat** [comment]
capture du 2017-07-03 15-08-34

**Comment by aduprat** [comment]
capture du 2017-07-03 15-44-07

File: src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapUIDFetchScenario.scala [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      |To: recipient@example.com
+      |Subject: test subject
+      |
+      |Test content
+      |abcdefghijklmnopqrstuvwxyz
+      |0123456789""".stripMargin).check(ok))
+
+  val populateInbox = repeat(numberOfMailInInbox)(pause(appendGracePeriod).exec(populateMailbox))
+
+  val uidFetch = exec(imap("uidFetch").uidFetch(MessageRanges(Range(1, numberOfMailInInbox)), AttributeList("BODY", "UID")).check(ok))
```
Comment (chibenwa) [inline src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapUIDFetchScenario.scala]: Could you remove "body"?

**Comment by chibenwa** [comment]
These results seems rather good... I would not be surprised all fits in fs cache and in cassandra memtable. Thus I would advocate it do not represent reality well...

**Pull request merged**